build.sh

```diff
@@ -1,84 +0,0 @@
-#!/usr/bin/env bash
-#
-# LPM | Author: Ben Marten
-# Copyright (c) 2017 Leanplum Inc. All rights reserved.
-#
-set -eo pipefail; [[ $DEBUG ]] && set -x
```

Comment:
This essentially means we just deprecated and broke lpm from being able to do an Android release. Which is fine as long as we are aware that this is a breaking change for lpm

Reply:
Yes. Notice this is on `master` and what we are doing is _replacing lpm_. On develop lpm continues to work (or not) as well as it used to.

Reply:
👍